docs/Elasticsearch-batchsource.md

```diff
@@ -26,6 +26,9 @@ see Elasticsearch for additional query examples. (Macro-enabled)
 
 **schema:** The schema or mapping of the data in Elasticsearch.
 
+**additionalProperties:** Additional properties to use with the es-hadoop client when reading the data, 
```

Comment:
Same here

Reply:
@nitinmotgi This is changed at both places.